CHANGES.md

```diff
@@ -1,6 +1,423 @@
+---------------------------------------------
+* made explicit 'module' declarations use strict error handling by default
+  (see http://chapel.cray.com/docs/1.16/technotes/errorHandling.html#relaxed-mode-for-explicit-modules)
+* made reduce intents preserve the reduction variable's initial value
+* added support for iterators that can yield void values
+  (see http://chapel.cray.com/docs/1.16/technotes/voidVariables.html#void-functions-and-iterators)
+* added a requirement that `deinit()` routines have parentheses
+* distinguished between functions returning 'void' values and non-returning fns
+  (see http://chapel.cray.com/docs/1.16/technotes/voidVariables.html)
+* extended return intent overloads to support value and 'const ref' overloads
+  (see 'Return Intent Overloads' in the 'Procedures' chapter of the spec)
+* made it an error to apply bitwise-not (~) to boolean values
+
+Syntactic/Naming Changes
+------------------------
+* single-statement 'local' blocks now require a 'do' keyword
+* renamed the 'Barrier' module to 'Barriers'
+
+New Features
+------------
+* added a 'defer' statement to aid with cleanup
+  (see 'The Defer Statement' in the 'Statements' chapter of the language spec)
+* added support for 'try'/'try!' expressions ('catch' only in statement form)
+  (see http://chapel.cray.com/docs/1.16/technotes/errorHandling.html#try-expressions)
+* added a 'prototype' keyword to relax checks for explicit module declarations
+  (see http://chapel.cray.com/docs/1.16/technotes/errorHandling.html#fatal-mode-for-implicit-and-prototype-modules)
+* added support for conditional 'local' statements
+  (see http://chapel.cray.com/docs/1.16/technotes/local.html#syntax)
+* added prototype support for choosing error strictness on a module granularity
+  (see http://chapel.cray.com/docs/1.16/technotes/errorHandling.html#strict-mode)
+* added support for reindex() on arrays to accept a list of ranges
+  (see http://chapel.cray.com/docs/1.16/builtins/internal/ChapelArray.html#ChapelArray.reindex)
+
+Feature Improvements
+--------------------
+* significantly improved the implementation of initializers
+  (see http://chapel.cray.com/docs/1.16/technotes/initializers.html)
+* significantly improved support for error-handling
+  (see http://chapel.cray.com/docs/1.16/technotes/errorHandling.html)
+* significantly improved the generality and robustness of 'forwarding' fields
+* improved the CHPL_UNWIND output to include more functions
+* made --print-callgraph print calls into non-user code
+* improved overload disambiguation for functions w/ partially generic arguments
+
+Removed Features
+----------------
+* removed support for the deprecated '=>' operator for creating array aliases
+* removed support for deprecated 'RandomStream' constructors
+* retired support for the deprecated CHPL_TASKS=muxed runtime option
+
+Standard Modules/Library
+------------------------
+* added 'throw'ing routines to 'IO', 'Regexp', 'FileSystem', 'Spawn', & 'Path'
```

Comment:
We also added 'throw'ing routines to HDFS and Buffers.

Reply:
HDFS is listed further down in the Packages section.  I either overlooked 'Buffers' or may have skipped it because I couldn't fit it into 80 columns (and didn't consider it as core for end-users as the others?).  I could also reword to say something like "to modules like IO, Regexp, FileSystem, Spawn, and others" or something like that.